src/pivot-table/hooks/use-cell-height.ts

```diff
@@ -0,0 +1,35 @@
+import { useMemo } from "react";
+import { LayoutService, StyleService } from "../../types/types";
+import { DEFAULT_ROW_HEIGHT } from "../constants";
+
+interface UseCellHeightProps {
+  styleService: StyleService;
+  layoutService: LayoutService;
+}
+
+interface UseCellHeight {
+  (args: UseCellHeightProps): {
+    lineClamp: number;
+    headerCellHeight: number;
+    contentCellHeight: number;
+  };
+}
+
+const LINE_HEIGHT_COEFFICIENT = 4 / 3;
+const fontSizeToRowHeight = (fontSize: string, lineClamp: number) =>
+  +(parseInt(fontSize, 10) * LINE_HEIGHT_COEFFICIENT * lineClamp).toFixed(2);
```

Comment:
Is this rounding from sn-table? If so, there might be a bug caused by it. Let's keep our eyes open if borders disappear for certain font sizes

Reply:
not actually, since we have `LINE_HEIGHT_COEFFICIENT` we sometimes end up in numbers like 48.3333333334. and those numbers were causing some issues in test => so i decided to only round it with 2 digits (if it was float number).

but sure, we can consider that possible bugs as well

Reply:
yeah there is something similar in the table, and I suspect we have a bug because of it. But if everything looks fine for now, no need to change anything id say

Reply:
I would have actually tought that `.toFixed(2)` helps to mitigate the floating precision issue